[DOCS] Adds Docker install option to model deployment (#2418)
elastic/stack-docs: docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

Eland encapsulates both the conversion of Hugging Face transformer models to
their TorchScript representations and the chunking process in a single Python
method; it is therefore the recommended import method. You can either install 

> the Python Eland client on your machine or use a Docker image to built Eland.

Reviewer: ```suggestion
the Python Eland client on your machine or use a Docker image to build Eland.
```

For more details, refer to https://github.com/elastic/eland#nlp-with-pytorch.

[discrete]
[[ml-nlp-import-docker]]
=== Import with Docker

IMPORTANT: To use the Docker container, you need to clone the Eland repository: 
https://github.com/elastic/eland

> If you want to use Eland without installing it, build the Docker container to 
> run the available scripts:

Reviewer: Its not obvious that the following command is run from the root of the cloned Eland repo.

```suggestion
If you want to use Eland without installing it, clone the Eland repository and from the root directory run the following to build the Docker container:
```

```bash
$ docker build -t elastic/eland .
```

You can now use the container interactively:

```bash
$ docker run -it --rm --network host elastic/eland
```

> Running installed scripts is also possible without an interactive shell:

Reviewer: ```suggestion
The `eland_import_hub_model` script can be run directly in the docker command:
```

```bash
docker run -it --rm elastic/eland \
    eland_import_hub_model \
      --url $ELASTICSEARCH_URL \
      --hub-model-id elastic/distilbert-base-uncased-finetuned-conll03-english \
      --start
```

Replace the `$ELASTICSEARCH_URL` with the URL for your {es} cluster. For 

> authentication purposes, include an administrator username and password in the 

Reviewer: Instead of the second sentence can you say something like "see the Authentication methods section for details"

Author: Addressed via [03f4002](https://github.com/elastic/stack-docs/pull/2418/commits/03f4002e8fa49dbf7ff7805a2c7996e252e3ca03).
